src/main/java/io/github/toomanybugs1/QuickFarm/Main.java

```diff
@@ -4,6 +4,9 @@
 import java.util.Collection;
 import java.util.List;
 
+import org.bukkit.*;
```

Comment:
I'd rather have all the Imports sorted alphabetically, and not having the Bukkit Imports divided in two chunks